homeassistant/components/deconz/__init__.py

```diff
@@ -157,6 +158,13 @@ def async_add_remote(sensors):
         deconz = hass.data[DOMAIN]
         if entity_id:
 
+            subpath = ''
+            if field:
```

Comment:
regex in service schema will take care of this. Just combine entity field and subfield